src/dodal/beamlines/i22.py

```diff
@@ -330,7 +385,7 @@ def panda3(
     )
 
 
-@skip_device()
+@skip_device(lambda: BL == _LAB_NAME)
```

Comment:
Add to panda2, panda3 also: don't need to skip those devices on i22 anymore, and we're certain that we're reasonably sure only deploying alongside compatible pandas.